lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -109,12 +116,27 @@ def get(self, request, *args, **kwargs):
         course_grade = CourseGradeFactory().read(request.user, course)
         courseware_summary = course_grade.chapter_grades.values()
 
+        verif_status = IDVerificationService.user_status(request.user)
+        verif_link = None
+        if verif_status['status'] is None:
+            verif_link = IDVerificationService.get_verify_location('verify_student_verify_now', course_id=course_key)
+        elif verif_status['status'] == 'must_reverify' or verif_status['status'] == 'expired':
+            verif_link = IDVerificationService.get_verify_location('verify_student_reverify', course_id=course_key)
+        verification_data = {
+            'expiration': verif_status['verification_expiry'],
+            'link': verif_link,
+            'status': verif_status['status'],
+            'status_date': verif_status['status_date'],
+        }
```

Comment:
Up to you, but rather than construct this object, you could just add the link to the verification_status object and then just pass that object into the serializer.